database/seeders/DatabaseSeeder.php

```diff
@@ -25,7 +26,8 @@ public function run(): void
         //     'email' => 'test@example.com',
         // ]);
 
-        CityFactory::new()->count(50)->create();
-        AirlineFactory::new()->count(20)->create();
+        CityFactory::new()->count(10)->create();
+        AirlineFactory::new()->count(5)->create();
+        FlightFactory::new()->count(50)->create();
```

Comment:
```suggestion
        CityFactory::times(10)->create();
        AirlineFactory::times(5)->create();
        FlightFactory::times(50)->create();
```
Same thing, but less code.